### PR TITLE
feat: adding alias to event payload

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**
Missing alias in event payload

https://oat-sa.atlassian.net/browse/AUT-16

### CODE REVIEW

**Review by pnal — APPROVED** [review]
LGTM, I'm not able to run/check locally but changes look harmless
**Please fix failing tests.**
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: model/publishing/delivery/tasks/RemoteTaskStatusSynchroniser.php [hunk 1]
```diff
@@ -183,11 +190,11 @@ private function callRemoteApi(string $url): array
      * @param string $deliveryUri
      * @param string $testUri
      */
-    private function triggerRemoteDeliveryCreatedEvent(?string $remoteDeliveryId, string $deliveryUri, string $testUri): void
+    private function triggerRemoteDeliveryCreatedEvent(?string $remoteDeliveryId, string $deliveryUri, string $testUri, ?string $alias): void
```
Comment (mike-ionut-mihai-sandu-tao) [inline model/publishing/delivery/tasks/RemoteTaskStatusSynchroniser.php:193]: `?string`
is php 7.4 version and i am bit reluctant that we have updated all environments to that version 
i would suggest to you to use the safe approach

**Review by mike-ionut-mihai-sandu-tao — COMMENTED** [review]
lgtm , left a comment about the strong type for that parameter

File: model/publishing/delivery/tasks/RemoteTaskStatusSynchroniser.php (hunk 1 above)
Comment (mike-ionut-mihai-sandu-tao) [reply model/publishing/delivery/tasks/RemoteTaskStatusSynchroniser.php:193]: that's not true,  verified again , good job

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**